src/gui/scaleChargesDialogFuncs.cpp

```diff
@@ -1,21 +1,19 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (c) 2024 Team Dissolve and contributors
 
+#include "gui/models/scaleChargesDialogModel.h"
 #include "gui/scaleChargesDialog.h"
+#include <QQmlContext>
+#include <QQuickItem>
+#include <QQuickWidget>
 
-ScaleChargesDialog::ScaleChargesDialog(QWidget *parent) { ui_.setupUi(this); }
-ScaleChargesDialog::~ScaleChargesDialog() {}
+ScaleChargesDialog::ScaleChargesDialog(QWidget *parent) : QDialog(parent)
+{
+    QQuickWidget *view = new QQuickWidget(QUrl("qrc:/dialogs/qml/ScaleChargesDialog.qml"), this);
```

Comment:
```suggestion
    QQuickWidget *view = new QQuickWidget(QUrl("qrc:/dialogs/qml/ScaleChargesDialog.qml"), this);
    view-setResizeMode(QQuickWidget::SizeRootObjectToView)
```

Tell QML to perform a resize when `view` is resized.